client/src/js/settings/components/Jobs/TaskField.js

```diff
@@ -9,19 +9,21 @@ export default class TaskField extends React.PureComponent {
     constructor (props) {
         super(props);
         this.state = {
+            name: this.props.name,
```

Comment:
Not necessary to add `name` to `state`. Putting `props` in `state` is almost always an anti-pattern.